frontend/src/zoomin.pug

```diff
@@ -0,0 +1,19 @@
+#zoomin
+  h1 {{ info.user.displayName }} ({{ info.user.name }})
```

Comment:
for pug files that belongs to a sub component, e.g. side window, can we place them in a subfolder?

Reply:
i wanna move summary and authorship out as well, i think i will do those outside, imma gonna do for this first